src/test/ui/attributes/doc-attr.stderr

```diff
@@ -1,14 +1,26 @@
-error: unknown `doc` attribute `as_ptr`
+error: `#![doc(test(...)]` is only allowed as a crate level attribute
   --> $DIR/doc-attr.rs:4:7
    |
+LL | #[doc(test(no_crate_inject))]
+   |       ^^^^^^^^^^^^^^^^^^^^^
+
+error: unknown `doc` attribute `as_ptr`
+  --> $DIR/doc-attr.rs:5:7
+   |
 LL | #[doc(as_ptr)]
    |       ^^^^^^
 
+error: `#![doc(test(...)]` is only allowed as a crate level attribute
+  --> $DIR/doc-attr.rs:9:12
+   |
+LL |     #![doc(test(no_crate_inject))]
+   |            ^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Is there any suggestions/actions that the user could make? I wonder why would someone do this. Note, I don't know about `#![doc(test(...))]`.

Reply:
I think the error is pretty explicit, no?

Reply:
I guess this is doctest but why would they do this in the first place?

Reply:
Because they want to add attributes to their doctests? XD